.rubocop.yml

```diff
@@ -27,3 +27,5 @@ Style/HashTransformKeys:
   Enabled: true
 Style/HashTransformValues:
   Enabled: false
+Rails/FindBy:
```

Comment:
Worth a separate commit as this is a config change?

Reply:
I've now stuck this into another commit